explorer/src/main/java/com/scalar/client/tool/explorer/Explorer.java

```diff
@@ -129,10 +121,13 @@ public JsonArray scan(String assetId, JsonObject condition) {
   }
 
   private JsonObject executeContract(String id, JsonObject argument) {
-    ClientServiceRequester requester =
-        () -> new ClientServiceResponse(clientService.executeContract(id, argument));
-    ClientServiceResponse response = fallback(requester);
-    return string2Json(response.getResult());
+    try {
+      ContractExecutionResult result = clientService.executeContract(id, argument);
+
+      return result.getResult().get();
+    } catch (ClientException clientException) {
+      throw clientException;
+    }
   }
 
   private JsonObject string2Json(String s) {
```

Comment:
It is not used any more ?

Reply:
Thank you for review change.
Unwanted methods removed in 135ba8e5358cdc9bfd02084406fd65ba68b3ddd4